c10/core/DispatchKeySet.h

```diff
@@ -132,4 +132,11 @@ static inline DispatchKey legacyExtractDispatchKey(DispatchKeySet s) {
   return s.highestPriorityTypeId();
 }
 
+// For backwards compatibility with XLA repository
+// (I don't want to fix this in XLA right now because there might be
+// more renaming coming in the future.)
+static inline DispatchKeySet XLA() {
+  return DispatchKeySet{DispatchKey::XLA, DispatchKey::XLAPreAutograd};
```

Comment:
I don't know if we should really treat this as BC with XLA repo or not. Maybe this ought to just be a permanent fixture (especially if we add more Autograd keys for other backends, it will be better to store these in a centralized place.)